source/_integrations/meteo_lt.markdown

```diff
@@ -0,0 +1,75 @@
+---
+title: Meteo.lt
+description: Instructions on how to integrate Lithuanian Hydrometeorological Service (Meteo.lt) within Home Assistant.
+ha_release: 2025.11
+ha_iot_class: Cloud Polling
+ha_category:
+  - Weather
+ha_codeowners:
+  - '@xE1H'
+ha_config_flow: true
+ha_domain: meteo_lt
+ha_platforms:
+  - weather
+ha_integration_type: integration
```

Comment:
```suggestion
ha_integration_type: service
```
In core, it is defined as "service". This is the classification we use in the docs.
https://github.com/home-assistant/core/pull/152948/files#diff-7b2bdd5938f9f3e1593d55199d7ca03014975edccfff38048736dc20ab6a0ad1R7